maro/simulator/scenarios/cim/business_engine.py

```diff
@@ -55,41 +59,38 @@ def __init__(self, event_buffer: EventBuffer, topology: str, start_tick: int, ma
         self._full_on_vessels: MatrixAttributeAccessor = None
         self._vessel_plans: MatrixAttributeAccessor = None
 
-        # read transfer cost factors
+        # Read transfer cost factors.
         transfer_cost_factors = self._config["transfer_cost_factors"]
```

Comment:
so we still need the factor in metric, right?

Reply:
We have removed cost from metrics, this could be deleted, also the configuration in config files. This can leave to me after this PR merged.